resources/assets/js/design.js

```diff
@@ -10,4 +10,9 @@ ready(() => {
     element.addEventListener('submit', preventDefault);
     element.addEventListener('click', preventDefault);
   });
+
+  document.getElementById('delete-form').addEventListener('submit', (event) => {
```

Comment:
```
Uncaught TypeError: document.getElementById(...) is null
    js design.js:14
    ready ready.js:6
    js design.js:3
    Webpack 4
[design.js:14:11](webpack://engelsystem/resources/assets/js/design.js)
    js design.js:14
    (Async: EventListener.handleEvent)
    ready ready.js:6
    js design.js:3
    Webpack 4
```

on any page which not the design page

```suggestion
  document.getElementById('delete-form')?.addEventListener('submit', (event) => {
```

Reply:
I would rather say that the `design.js` file should not be loaded on non-design pages...

Reply:
Thats an idea that can be implemented eventually but at the moment that's not the case :D

Reply:
& its fixed